backend/src/estark/mod.rs

```diff
@@ -158,8 +185,16 @@ impl<'a, F: FieldElement> EStarkFilesCommon<'a, F> {
         }
 
         // Write the stark struct JSON.
+        let hash_type = match self.proof_type {
```

Comment:
I don't understand the hash type. GL and BN are not hash functions? Are these different variants of Poseidon?

Reply:
Hash_type is just the name of the field in the StarkInfo struct, and these are the values it accepts